testUtils/consoleMocker.js

```diff
@@ -16,5 +16,9 @@ export const ConsoleWritesForTesting = (() => {
 
 export const ConsoleMocker = (() => {
   const log = ConsoleWritesForTesting.addLog;
-  return { log };
+  const info = ConsoleWritesForTesting.addLog;
+  const debug = ConsoleWritesForTesting.addLog;
+  const warn = ConsoleWritesForTesting.addLog;
+  const error = ConsoleWritesForTesting.addLog;
+  return { log, info, warn, error, debug };
```

Comment:
We have the same set of tests for each log level. It looks like we mock each of the log functions to the same function.
So we actually test the same thing in every test, isn't it?

Reply:
Ammmm yes, but its black-boxed test
we cant assume they using the same function.